db/migrate/20201114130315_create_podcast_ownerships.rb

```diff
@@ -0,0 +1,9 @@
+class CreatePodcastOwnerships < ActiveRecord::Migration[6.0]
+  def change
+    create_table :podcast_ownerships do |t|
+      t.references :podcast, index: true, foreign_key: true
+      t.references :user, index: true, foreign_key: true
```

Comment:
can you add `null: false` here? Unfortunately by default [add_reference](https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference) defaults to `null: true`

Reply:
don't forget `null: false`